lib/util.js

```diff
@@ -394,6 +404,23 @@ function formatValue(ctx, value, recurseTimes, ln) {
     return ctx.stylize('null', 'null');
   }
 
+  const now = getNow();
```

Comment:
Can't we just pass the previous value to `process.hrtime()` to get a time difference and then compare that value instead?

Reply:
Yes, but that requires a second property on the `ctx` object and I would like to prevent that.